From af94ae30d03668f0f4c93852d8e2d1f88ed04c62 Mon Sep 17 00:00:00 2001
From: Skylab tests cros image roller
 <skylab-test-cros-roller@chops-service-accounts.iam.gserviceaccount.com>
Date: Fri, 10 Jun 2022 00:38:54 +0000
Subject: [PATCH 01/59] [skylab_tests] Update skylab tests cros img version

This CL will update cros image version for skylab tests. CROS_BOARD_LKGM
is updated according to chromeos/CHROMEOS_LKGM; CROS_BOARD_DEV,
CROS_BOARD_BETA and CROS_BOARD_STABLE are updated according to Omaha.

If this CL caused regressions, please revert and pause the autoroller at
https://luci-scheduler.appspot.com/jobs/chrome/lacros-skylab-tests-cros-img-roller
Also please file a bug to OS>LaCrOS>Partner, and CC svenzheng@chromium.org,
yjt@google.com.


R=rubber-stamper@appspot.gserviceaccount.com

Requires-Testing: True
Change-Id: I5b29e1b2b6de832dc51758037519f76f3f8f7ebd
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3698986
Bot-Commit: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Commit-Queue: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Auto-Submit: skylab-test-cros-roller@chops-service-accounts.iam.gserviceaccount.com <skylab-test-cros-roller@chops-service-accounts.iam.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/5005@{#1160}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 testing/buildbot/variants.pyl | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/testing/buildbot/variants.pyl b/testing/buildbot/variants.pyl
index d6a822d49671..389bb3539547 100644
--- a/testing/buildbot/variants.pyl
+++ b/testing/buildbot/variants.pyl
@@ -919,8 +919,8 @@
   'CROS_ATLAS_RELEASE_BETA': {
     'skylab': {
       'cros_board': 'atlas',
-      'cros_chrome_version': '103.0.5060.33',
-      'cros_img': 'atlas-release/R103-14816.41.0',
+      'cros_chrome_version': '103.0.5060.37',
+      'cros_img': 'atlas-release/R103-14816.49.0',
     },
     'enabled': False,
     'identifier': 'ATLAS_RELEASE_BETA',
@@ -955,8 +955,8 @@
   'CROS_EVE_RELEASE_BETA': {
     'skylab': {
       'cros_board': 'eve',
-      'cros_chrome_version': '103.0.5060.33',
-      'cros_img': 'eve-release/R103-14816.41.0',
+      'cros_chrome_version': '103.0.5060.37',
+      'cros_img': 'eve-release/R103-14816.49.0',
     },
     'enabled': False,
     'identifier': 'EVE_RELEASE_BETA',
@@ -1000,8 +1000,8 @@
   'CROS_HANA_RELEASE_BETA': {
     'skylab': {
       'cros_board': 'hana',
-      'cros_chrome_version': '103.0.5060.33',
-      'cros_img': 'hana-release/R103-14816.41.0',
+      'cros_chrome_version': '103.0.5060.37',
+      'cros_img': 'hana-release/R103-14816.49.0',
     },
     'enabled': False,
     'identifier': 'HANA_RELEASE_LKGM',
@@ -1036,8 +1036,8 @@
   'CROS_JACUZZI_RELEASE_BETA': {
     'skylab': {
       'cros_board': 'jacuzzi',
-      'cros_chrome_version': '103.0.5060.33',
-      'cros_img': 'jacuzzi-release/R103-14816.41.0',
+      'cros_chrome_version': '103.0.5060.37',
+      'cros_img': 'jacuzzi-release/R103-14816.49.0',
     },
     'enabled': False,
     'identifier': 'JACUZZI_RELEASE_BETA',
@@ -1081,8 +1081,8 @@
   'CROS_OCTOPUS_RELEASE_BETA': {
     'skylab': {
       'cros_board': 'octopus',
-      'cros_chrome_version': '103.0.5060.33',
-      'cros_img': 'octopus-release/R103-14816.41.0',
+      'cros_chrome_version': '103.0.5060.37',
+      'cros_img': 'octopus-release/R103-14816.49.0',
     },
     'enabled': False,
     'identifier': 'OCTOPUS_RELEASE_BETA',
-- 
2.37.0

